format

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+black zen_queries
+flake8 zen_queries
```

Comment:
Issue: `flake8`s default entrypoint _checks_ for errors, rather than _fixing_ them. To fix them, you'd need something like [`fix8`](https://github.com/PeterJCLaw/fix8) or [`flake8_autofix`](https://github.com/sciencefreak500/flake8_autofix)

Reply:
Good spot, have removed.